x-pack/plugins/maps/public/components/layer_panel/join_editor/resources/metrics_expression.js

```diff
@@ -82,14 +86,23 @@ export class MetricsExpression extends Component {
         button={
           <EuiExpression
             onClick={this._togglePopover}
-            description={metricExpressions.length > 1 ? 'and use metrics' : 'and use metric'}
+            description={metricExpressions.length > 1 ? i18n.translate('xpack.maps.layerPanel.metricsExpression.useMetrics', {
```

Comment:
This is pretty complicated logic for JSX how about moving this into a const in the javascript code above to keep the JSX more readable